src/types.ts

```diff
@@ -305,7 +325,12 @@ export interface DomainMetadata {
   animation_url: string | undefined;
   name: string;
   description: string;
+  stakingRequests: "disabled" | "enabled" | undefined;
+  isBiddable: boolean | undefined;
+  gridViewByDefault: boolean;
```

Comment:
Needs to be `boolean | undefined` since we have many domains out there which don't have this property set

Reply:
You could also do `gridViewByDefault?: boolean` since they are the same